### DISCUSSION TITLE
Once purchased, is this "open source"?

### DISCUSSION THREAD

**Discussion by dhamelin** [KirillKuzyk/Google-Play-License-Check#13] [Q&A] [answered]
Hi, 

Once we have purchased the asset, do we have access to the full source code? I have seen so many asset/plugins being abandoned over the years, and it's really useful for devs to have access to source to fix issues. So, if not yet "open source", please consider it!

Thank you.

**Comment by KyryloKuzyk** [comment, marked as answer]
Hi,

Yes, the full source code is available after the purchase. I also believe all Asset Store plugins should come will the source code.
